### PR TITLE
Add Pointers documentation

### PR DESCRIPTION
**Pull request by AndrewMeadows**

### CODE REVIEW

File: 06.api-reference/28.picks/docs.md [hunk 1]
```diff
@@ -106,16 +104,16 @@ An intersection result for a Ray Pick.
...
 Adds a new top-level menu. 
 
 **Parameters**
 
 |Name|Type|Description|
 |----|----|-----------|
-|type|PickType |A PickType that specifies the method of picking to use.|
-|properties|[PickProperties](#p1)|A [PickProperties](#p1) object, containing all the properties for initializing this Pick|
+|type|[PickType](#PickType) |A PickType that specifies the method of picking to use.|
+|properties|[PickProperties](#PickProperties)|A [PickProperties](#PickProperties) object, containing all the properties for initializing this Pick|
```
Comment (MiladNazeri) [inline 06.api-reference/28.picks/docs.md]: . and the end of Pick for consistency

File: 06.api-reference/28.picks/docs.md [hunk 2]
```diff
@@ -135,7 +133,7 @@ Disables a Pick.
 |uid|number|The ID of the Pick, as returned by Picks.createPick.|
 
 
-### `enablePick(uid)`<a id="m3"></a>
+### `enablePick(uid)`<a id="enablePick"></a>
```
Comment (MiladNazeri) [inline 06.api-reference/28.picks/docs.md]: Is there a difference between uid and uuid?

File: 06.api-reference/28.picks/docs.md [hunk 3]
```diff
@@ -200,7 +198,7 @@ Checks if a Pick is associated with the system mouse.
 |`false`|boolean|If the Pick is not a Mouse ray Pick|
```
Comment (MiladNazeri) [inline 06.api-reference/28.picks/docs.md]: . for consistency

Comment (MiladNazeri) [inline 06.api-reference/28.picks/docs.md]: . for consistency

File: 06.api-reference/28.picks/docs.md [hunk 4]
```diff
@@ -106,16 +104,16 @@ An intersection result for a Ray Pick.
 
 ## Methods <a id="methods"></a>
 
-### `createPick(type, properties)`<a id="m1"></a>
+### `createPick(type, properties)`<a id="createPick"></a>
 
 Adds a new top-level menu.
```
Comment (MiladNazeri) [inline 06.api-reference/28.picks/docs.md]: A little confused by what is meant by "Adds a new top-level menu"

File: 06.api-reference/47.pointers/docs.md [hunk 5]
```diff
@@ -0,0 +1,273 @@
...
+
+**Returns**
+
+|Type|Description|
+|----|-----------|
+|number|The ID of the Pointer created. Used for managing the Pointer. `0` if invalid. |
+
+
+
+    /**jsdoc
```
Comment (MiladNazeri) [inline 06.api-reference/47.pointers/docs.md]: Don't think this was meant to be here

File: 06.api-reference/47.pointers/docs.md [hunk 6]
```diff
@@ -0,0 +1,273 @@
...
+
+|Name|Type|Description|
+|----|----|-----------|
+| `id` | number | The ID of the Pointer, as returned by [createPointer()](#createPointer). |
+
+**Returns**
+
+|Type|Description| 
+|----|-----------| 
+| PickResult | The most recent intersection result.  This will be slightly different for different PickTypes.  See [Picks.RayPickResult](../..//picks/RayPickResult) and [Picks.StylusPickResult](../..//picks/StylusPickResult) |
```
Comment (MiladNazeri) [inline 06.api-reference/47.pointers/docs.md]: . for consistency

File: 06.api-reference/47.pointers/docs.md [hunk 7]
```diff
@@ -0,0 +1,273 @@
...
+### `setIncludeItems(id, includeItems)` <a id="setIncludeItems"></a>
+Sets a list of Entity IDs, Overlay IDs, and/or Avatar IDs to include during intersection, instead of intersecting with everything.
+Stylus Pointers <b>only</b> intersect with objects in their include list.
+
+**Parameters**
+
+|Name|Type|Description|
+|----|----|-----------|
+| `id` | number | The ID of the Pointer, as returned by [createPointer()](#createPointer). |
+| includeItems | Uuid[] | | A list of IDs to include. |
```
Comment (MiladNazeri) [inline 06.api-reference/47.pointers/docs.md]: There is an extra column after Uuid and the description isn't being rendered.

File: 06.api-reference/47.pointers/docs.md [hunk 8]
```diff
@@ -0,0 +1,273 @@
...
+
+### `setIgnoreItems(id, ignoreItems)` <a id="setIgnoreItems"></a>
+Sets a list of Entity IDs, Overlay IDs, and/or Avatar IDs to ignore during intersection.  Not used by Stylus Pointers.
+
+**Parameters**
+
+|Name|Type|Description|
+|----|----|-----------|
+| `id` | number | The ID of the Pointer, as returned by [createPointer()](#createPointer). |
+| ignoreItems | Uuid[] | | A list of IDs to ignore. |
```
Comment (MiladNazeri) [inline 06.api-reference/47.pointers/docs.md]: Extra column after Uuid[] and the description isn't being rendered

File: 06.api-reference/47.pointers/docs.md [hunk 9]
```diff
@@ -0,0 +1,273 @@
...
+Not used by Stylus Pointers.
+
+**Parameters**
+
+|Name|Type|Description|
+|----|----|-----------|
+| `id` | number | The ID of the Pointer, as returned by [createPointer()](#createPointer). |
+| objectID | Uuid | The ID of the object to which to lock on. |
+| isOverlay | boolean | False for entities or avatars, true for overlays. |
+| offsetrMat | Mat4 | The offset matrix to use if you do not want to lock on to the center of the object. |
```
Comment (MiladNazeri) [inline 06.api-reference/47.pointers/docs.md]: typo in name for "offsetrMat"

File: 06.api-reference/28.picks/docs.md [hunk 10]
```diff
@@ -124,53 +122,53 @@ Adds a new top-level menu.
 |The ID of the pick created|number|Used for managing the Pick. `0` if invalid. |
 
 
-### `disablePick(uid)`<a id="m2"></a>
+### `disablePick(id)`<a id="disablePick"></a>
```
Comment (AndrewMeadows) [inline 06.api-reference/28.picks/docs.md]: Uuid = universally unique id (128 bits randomly selected --> the odds of a collision is so small we ignore it)

id = 32-bit integer id.  It is unique within its context, but not universally so.

File: 06.api-reference/28.picks/docs.md (hunk 4 above)
Comment (AndrewMeadows) [reply 06.api-reference/28.picks/docs.md]: This was a typo from the previous author.

**Pull request merged**